src/plugin.ts

```diff
@@ -24,6 +24,11 @@ export function routerPlugin(options: RouterPluginOptions = {}): KeaPlugin {
           pathFromRoutesToWindow: (path) => path,
           pathFromWindowToRoutes: (path) => path,
           options,
+          beforeUnloadInterceptors: new Set(),
+          stateCount:
```

Comment:
Nit: `historyStateCount` is likely more descriptive. I'm left wondering "what state".